firefox_ui_tests/remote/security/test_ssl_status_after_restart.py

```diff
@@ -0,0 +1,123 @@
+                'identity': 'Mozilla Corporation',
+                'type': 'verifiedIdentity'
+            },
+            {
+                'url': 'https://ssl-ov.mozqa.com/',
+                'identity': '',
+                'type': 'verifiedDomain'
+            }
+        ]
+
+        # Set browser to restore previous session
+        self.prefs.set_pref('browser.startup.page', 3)
+
+        self.identity_popup = self.browser.navbar.locationbar.identity_popup
+
+    def tearDown(self):
+        try:
+            self.identity_popup.close(force=True)
+            self.browser.tabbar.close_all_tabs([self.browser.tabbar.tabs[0]])
+        except NoSuchElementException:
+            # TODO: A NoSuchElementException may be thrown here when the test is skipped
+            # as under xvfb.
+            pass
+        finally:
+            FirefoxTestCase.tearDown(self)
+
+    @skip_if_e10s
+    @skip_under_xvfb
+    # Bug 995801: Test certificate status after browser restart
+    def test_ssl_status_after_restart(self):
+        for item in self.test_data:
+            with self.marionette.using_context('content'):
+                self.marionette.navigate(item['url'])
+            self.verify_certificate_status(item)
+            self.browser.tabbar.open_tab()
+
+        '''
+        self.restart()
+
+        i = 0
+        for item in self.test_data:
+            self.browser.tabbar.switch_to(i)
+            self.verify_certificate_status(item)
+            i = i + 1
+        '''
+
+    def verify_certificate_status(self, item):
+        url = item['url']
+
+        # Check the favicon
+        # TODO: find a better way to check, e.g., mozmill's isDisplayed
+        self.assertEqual(self.browser.navbar.locationbar.favicon.get_attribute('hidden'), 'false',
+                         'Lock icon is visible in identity box for ' + url)
+
+        self.identity_popup.box.click()
+        Wait(self.marionette).until(lambda _: self.identity_popup.is_open)
+
+        # Check the type shown on the idenity popup doorhanger
+        self.assertEqual(self.identity_popup.popup.get_attribute('className'),
+                         item['type'],
+                         'Extended certificate is verified for ' + url)
+
+        # Check the identity label
+        self.assertEqual(self.identity_popup.organization_label.get_attribute('value'),
+                         item['identity'],
+                         'Identity name is correct for ' + url)
+
+        # Get the information from the certificate
+        cert = self.browser.tabbar.selected_tab.certificate
+
+        # Open the Page Info window by clicking the More Information button
+        page_info = self.browser.open_page_info_window(
+            lambda _: self.identity_popup.more_info_button.click())
+
+        if item['identity'] != '':
+            owner = cert['organization']
+        else:
+            owner = page_info.get_property('securityNoOwner')
```

Comment:
This is a change in behavior compared to the mozmill tests. Can you explain why this is necessary?